rotkehlchen/exchanges/binance.py

```diff
@@ -530,6 +537,66 @@ def _query_spot_balances(
 
         return balances
 
+    def _query_funding_balances(
+            self,
+            balances: defaultdict[AssetWithOracles, Balance],
+    ) -> defaultdict[AssetWithOracles, Balance]:
```

Comment:
It is missing the docstring. If you can please also link to the docs for this endpoint